FWCore/Utilities/src/ObjectWithDict.cc

```diff
@@ -74,9 +76,13 @@ namespace edm {
     }
 
     if (to.hasBase(from)) { // down cast
+#ifndef _LIBCPP_VERSION
       // use the internal dynamic casting of the compiler (e.g. libstdc++.so)
       void* address = abi::__dynamic_cast(address_, static_cast<abi::__class_type_info const*>(&from.typeInfo()), static_cast<abi::__class_type_info const*>(&to.typeInfo()), -1);
```

Comment:
@gartung could you comment on why you need to disable this part?
From https://libcxxabi.llvm.org/spec.html
`void* __dynamic_cast(const void* __src_ptr, const __class_type_info* __src_type, const __class_type_info* __dst_type, ptrdiff_t __src2dst);`
From GNU:
```
  void*
  __dynamic_cast(const void* __src_ptr, // Starting object.
		 const __class_type_info* __src_type, // Static type of object.
		 const __class_type_info* __dst_type, // Desired target type.
		 ptrdiff_t __src2dst); // How src and dst are related.
```
In GNU it's under `namespace abi = __cxxabiv1;` and same applies for LLVM implementation: `namespace abi = __cxxabiv1;`